src/Field.jsx

```diff
@@ -0,0 +1,63 @@
+import {
+    Grid,
+    InputAdornment,
+    MenuItem,
+    Select,
+    TextField,
+    Typography
+} from '@material-ui/core';
+import ReactDOM from "react-dom";
+import React from "react";
+
+
+export default function Field (props) {
+    function handleChange(event) {
+        props.updateParent(props.metadata.name, event.target.value, !props.isValueValid(event.target.value));
+    }
+    switch (props.metadata.type) {
+        case 'Binary' :
+        case 'binary' :
```

Comment:
Why both `Binary` and `binary`?

Reply:
Historical reasons. Do you want the duplicates removed?

Reply:
If they are not in use, and since this is the first version, there's no use for the duplicates. Note that there's another one below for `list`.